generators/app/templates/src/app/components/SearchBar/index.js

```diff
@@ -22,17 +22,17 @@ function SearchBarContainer({
 }
 
 SearchBarContainer.propTypes = {
-  textButtonSearch: PropTypes.string,
-  className: PropTypes.string,
-  formClassName: PropTypes.string,
+  handleSubmit: PropTypes.func.isRequired,
   buttonClassName: PropTypes.string,
   children: PropTypes.node,
-  handleSubmit: PropTypes.func.isRequired
+  className: PropTypes.string,
```

Comment:
I think `className` propTypes isn't necessary

Reply:
Why? I'm using it in this components's parent div